data-sources/ansible-automation-controller/config.yml

```diff
@@ -6,3 +6,4 @@ install:
   primary:
     link:
       url: https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/ansible-controller-integration/
+icon: logo.png
```

Comment:
This logo is going to look terrible when it's shrunk down

Reply:
Good call, updated the icon